README.md

```diff
@@ -101,6 +101,27 @@ public class Main {
 }
 ```
 
+## Reusing http clients
+
+Make sure you don't create a new provider every time you want to use one. Instead, you should reuse existing instance.
+This way you reuse connections and thread pools. You can read more about it [here](https://square.github.io/okhttp/4.x/okhttp/okhttp3/-ok-http-client/#okhttpclients-should-be-shared).
```

Comment:
I'd move the lines about reusing connection pools etc. to the OkHttpService docs.